src/lib.rs

```diff
@@ -9,6 +9,8 @@
 #![deny(warnings)]
 #![forbid(unsafe_code)]
 #![cfg_attr(any(target_arch = "wasm32", not(feature = "std")), no_std)]
+// TODO: remove when boxed
+#![allow(clippy::result_large_err)]
```

Comment:
I think we did box already, so this can go, right?

Reply:
Yes!